app/components/ExternalLinksBlock/ExternalLinksBlockContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import _ from 'lodash';
+import ExternalLinksBlock from './ExternalLinksBlock';
+
+export default class ExternalLinksBlockContainer extends React.Component {
+  getExternalLinks() {
```

Comment:
They've never needed to be bound, except for event handlers. I've been asking for them to removed. They don't do anything, as far as I can tell, except create a new, identical copy of the original method. It can be hard to debug whether test spies and stubs are wrapping the original or the copy.

Re. event handlers, I noticed that the Facebook codemod, which we've used to generate a few ES6 components, prefers to bind in the JSX render: `onClick={this.handleClick.bind(this_}`

Reply:
Huh, today I learned. Thanks @eatyourgreens, I must have misunderstood what was required when React moved away from `createClass`

Reply:
I've only been picky about this since we upgraded to enzyme 3. Tests that spy on `instance.method` have problems now if `method` has been bound to a new context.

They've always puzzled me since class methods are bound to instances of the class by default. Event handlers and promise callbacks invoke functions with a new `this` context, if you don't bind them. Beyond those two cases, I think it should be okay not to use them.

Reply:
[I think this is what caught me out](https://reactjs.org/blog/2015/01/27/react-v0.13.0-beta-1.html#autobinding):

> React.createClass has a built-in magic feature that bound all methods to this automatically for you. This can be a little confusing for JavaScript developers that are not used to this feature in other classes, or it can be confusing when they move from React to other classes.

> **Therefore we decided not to have this built-in into React’s class model.** You can still explicitly prebind methods in your constructor if you want.

(emphasis mine)

There wasn't a mention on when you would and wouldn't need to do it, so I probably assumed it needed to be done ¯\_(ツ)_/¯